feature/sessions/src/commonMain/kotlin/io/github/droidkaigi/confsched/sessions/components/SearchFilterRow.kt

```diff
@@ -176,25 +205,40 @@ private fun <T> FilterDropdown(
 
         DropdownMenu(
             expanded = expanded,
-            onDismissRequest = { expanded = false },
+            onDismissRequest = {
+                if (isMultiSelectMode) onMultiSelectFinished(selectedItems.toList())
+                expanded = false
+                isMultiSelectMode = false
+            },
         ) {
             items.forEach { item ->
-                DropdownMenuItem(
-                    modifier = Modifier.testTag(DropdownFilterChipTestTagPrefix.plus(item)),
-                    leadingIcon = {
-                        if (selectedItems.contains(item)) {
-                            Icon(
-                                imageVector = Icons.Default.Check,
-                                contentDescription = null,
-                            )
-                        }
-                    },
-                    text = { Text(itemLabel(item)) },
-                    onClick = {
-                        onItemSelected(item)
-                        expanded = false
-                    },
-                )
+                Row(
+                    modifier = Modifier
+                        .testTag(DropdownFilterChipTestTagPrefix.plus(item))
+                        .fillMaxWidth()
+                        .heightIn(min = 48.dp)
+                        .padding(horizontal = 12.dp, vertical = 8.dp)
+                        .combinedClickable(
+                            onClick = {
+                                onItemSelected(item)
+                                if (!isMultiSelectMode) {
+                                    expanded = false
+                                }
+                            },
+                            onLongClick = {
+                                if (!isMultiSelectMode) isMultiSelectMode = true
+                                haptics.performHapticFeedback(HapticFeedbackType.LongPress)
+                                onItemLongPress?.invoke(item)
+                                onItemSelected(item)
+                            },
+                        ),
+                    verticalAlignment = Alignment.CenterVertically,
+                ) {
+                    if (item in selectedItems) {
+                        Icon(Icons.Default.Check, contentDescription = null, modifier = Modifier.padding(end = 12.dp))
+                    }
```

Comment:
Hiding the icon entirely when unselected might cause the view width to shift compared to the existing behavior, which could feel janky. Maybe controlling the alpha instead would be smoother.

https://github.com/user-attachments/assets/d3f0d5f4-3db9-43c5-a706-a203aec4860f